### ISSUE TITLE
[BUG] subtitles not working for some weird reason

### ISSUE THREAD

**Issue by KurdishSage-1** [jhomlala/betterplayer#691]
i've been wrapping my head around this for a couple days, betterplayer works with some subtitle files, and doesn't work for some subtitle files, 

i have 2 identical subtitle files that are absolutely identical (except one of the files is a couple bytes smaller than the other one,
the file with smaller bytes does not work, but the other one works.

i've also tested a movie subtitle file that has extra lines between the texts, it's not working; so i deleted the extra lines with a python script, that one doesn't work either.

[here's the subtitle files](https://drive.google.com/drive/folders/1FfL8ib6p24YL6IViXt2XVNaLwOUwR4iE?usp=sharing)

i'm using this code in my flutter app
```subtitles: BetterPlayerSubtitlesSource.single(
          type: BetterPlayerSubtitlesSourceType.file,
          name: "Default",
          selectedByDefault: true,
          url: "/storage/emulated/0/Download/wawa.srt",
        ),
```

what i also tried, is uploading the srt file in to internet and use `type: BetterPlayerSubtitlesSourceType.network` to read the subtitle file, that didn't had any effects. i also tested numerous amount of subtitle files (from opensubtitles.org) and none of them are working in betterplayer

what causes this problem?

i'm using Redmi Note 6 Pro, and betterplayer 0.0.73 (also tried 0.0.72, it affected nothing).

**Comment by jhomlala** [comment]
@KurdishSage-1 I've checked these subtitles on my side:
* `not working 1.srt`
![flutter_01](https://user-images.githubusercontent.com/1329033/132390091-917425d8-25e7-4ae7-a3f5-8ad026045566.png)

* `not working 2 (without extra lines).srt
![flutter_02](https://user-images.githubusercontent.com/1329033/132390649-bbba837d-c55f-40ac-a44e-2bcae8430e9f.png)

* `not working 2.srt` is invalid file - these empty lines should be removed.

So basically these subtitles are working fine. You can check in `subtitles` page in example app.

**Comment by jhomlala** [comment]
No response from author. Closing issue.
